nav fixes + new color palette

moonitini opened the pull request.

moonitini committed:
nav fixes + new color palette

milkcee12 commented on src/lib/components/Button.svelte:
```diff
@@ -62,8 +62,8 @@
 	}
-	.bg-earth-green {
-		background-color: $earth-green;
+	.bg-green {
```
Looks like you use lime in a lot of other places besides here, maybe keep the naming consistent and name the selector `.bg-lime`.

milkcee12 commented on src/lib/components/Header.svelte:
```diff
@@ -3,7 +3,8 @@
     h2 {
         font-family: 'Lilita One', sans-serif;
-        color: $text-pink;
+        // color: $text-pink;
```
nit: remove commented code

milkcee12 commented on src/lib/components/NavbarMobile.svelte:
```diff
@@ -59,23 +59,23 @@
 			<ul>
 				<li class:active={activeSection == 'home'}>
-					<a href="#" on:click={scrollToTop}>Home</a>
+					<a href="#/" on:click={scrollToTop}>Home</a>
```
Just checking, is there supposed to be an extra slash here?

moonitini replied:
the terminal was throwing errors when I had just the '#' and changing it to '#/" fixed them? I can change it back if you want?

milkcee12 commented on src/lib/components/Button.svelte:
```diff
@@ -3,7 +3,7 @@
 <script lang="ts">
 	type TiltRange = 'left' | 'right' | 'none';
-	type ButtonBackgroundColors = 'blue' | 'pink' | 'gold' | 'earth-green';
+	type ButtonBackgroundColors = 'blue' | 'pink' | 'gold' | 'green';
```
Would also need to update the choices here to be `'lime'` instead of `'green'`.

milkcee12 commented on src/lib/components/sections/HeroSection.svelte:
```diff
@@ -14,12 +14,12 @@
 
 		<!-- [BEGIN] Hackathon application links -->
-		<Button
+		<!-- <Button
```
nit: Can you move this `Button` code to above line 14?

milkcee12 commented on src/lib/components/sections/Schedule.svelte:
Same here- You should be able to leave this file uncommented- as long as the relevant links are commented out on the navbar files it should be fine.

milkcee12 commented on src/lib/components/sections/Keynote.svelte:
You should be able to leave this file uncommented- as long as the relevant links are commented out on the navbar files it should be fine.

milkcee12 commented on src/lib/components/sections/Sponsors.svelte:
Same here, leave this file uncommented

moonitini replied:
<img width="756" alt="Screenshot 2024-10-09 at 1 19 52 PM" src="https://github.com/user-attachments/assets/d9fe4426-215e-4253-9d65-19b9d6d09e14">

Getting these warnings when I uncomment the css-- is that okay?

milkcee12 replied:
yeah that's fine! if they're bothering you I'd just delete them tbh and add them as needed in the future

milkcee12 commented on src/lib/scss/_variables.scss:
```diff
@@ -4,12 +4,22 @@ $blue: #BFE2FF;
 $brown: #401E0A;
 $light-brown: #5A3019;
-$pink: #FFB5CB;
+// $pink: #FFB5CB;
```
feel free to remove unused colors :)

milkcee12 commented on src/routes/+page.svelte:
Nice!!

milkcee12 commented on src/routes/+layout.svelte:
```diff
@@ -48,11 +48,11 @@
-					color: $brown;
-					font-size: 2rem;
-				}
+				// p {
```
I don't think we need this, you can just delete the entire commented code block.

milkcee12 reviewed (commented):
Looks mostly good, just code quality nits!

moonitini committed:
updated hero image, fixed code quality nits

- I left the tier 1 and tier 2 css commented because it was throwing errors at me when I uncommented them
- Navbar home href is '/#'
